app/mailers/prison_mailer.rb

```diff
@@ -28,7 +28,14 @@ def booking_request_email(visit, token)
     user = visit.visitors.find(&:email).email
     @token = token
 
-    mail(from: sender, reply_to: user, to: recipient, subject: "Visit request for #{@visit.prisoner.full_name} on #{Date.parse(@visit.slots.first.date).strftime('%A %e %B')}")
+    mail(from: sender,
+         reply_to: user,
+         to: recipient,
+         subject: default_i18n_subject(
+           full_name: @visit.prisoner.full_name,
+           date: Date.parse(@visit.slots.first.date).strftime('%A %e %B')
+         )
+        )
```

Comment:
I tried it, but then rubocop will say "Space inside parentheses detected." so what I can suggest is 

```ruby
mail(
      from: sender,
      reply_to: user,
      to: recipient,
      subject: default_i18n_subject(
        full_name: @visit.prisoner.full_name,
        date: Date.parse(@visit.slots.first.date).strftime('%A %e %B')
      )
    )
```